### ISSUE TITLE
Missing d.ts files in npm package

### ISSUE THREAD

**Issue by lchsam** [samdenty/console-feed#13]
Hi,

I believe you forgot to include d.ts files in your latest npm package.
I looked through every folder in the console-feed node_module and it doesn't seem to have it.

Here's what I have for the latest console-feed npm package.
```
console-feed
├── lib
│   ├── Component
│   │   ├── devtools-parser
│   │   │   ├── format-message.js
│   │   │   ├── index.js
│   │   │   └── string-utils.js
│   │   ├── elements.js
│   │   ├── index.js
│   │   ├── Message.js
│   │   ├── message-parsers
│   │   │   ├── Error.js
│   │   │   ├── Formatted.js
│   │   │   └── Object.js
│   │   ├── react-inspector
│   │   │   ├── elements.js
│   │   │   ├── index.js
│   │   │   └── util.js
│   │   └── theme
│   │       ├── default.js
│   │       └── index.js
│   ├── definitions
│   │   └── Methods.js
│   ├── Hook
│   │   ├── construct.js
│   │   ├── index.js
│   │   ├── parse
│   │   │   ├── GUID.js
│   │   │   ├── index.js
│   │   │   └── methods
│   │   │       ├── assert.js
│   │   │       ├── count.js
│   │   │       └── timing.js
│   │   └── store
│   │       ├── actions.js
│   │       ├── dispatch.js
│   │       ├── reducer.js
│   │       └── state.js
│   ├── index.js
│   ├── Transform
│   │   ├── arithmetic.js
│   │   ├── Function.js
│   │   ├── HTML.js
│   │   ├── index.js
│   │   ├── Map.js
│   │   └── replicator
│   │       └── index.js
│   └── Unhook
│       └── index.js
├── package.json
└── README.md
```

Thanks,
Sam L.

**Comment by bodinsamuel** [comment]
+1

**Comment by orlandodemauro** [comment]
@samdenty first of all thank you so much for this great job. I'm wondering if there will be a new release with this issue fixed. I'm importing the library in typescript and i'm getting the following error:


```
Could not find a declaration file for module 'console-feed'. '/Users/orlandodemauro/code/flowable-insight/frontend/vendor/flowable-steps/node_modules/console-feed/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/console-feed` if it exists or add a new declaration (.d.ts) file containing `declare module 'console-feed';`
```

**Comment by SpencerSharkey** [comment]
+1

**Comment by samdenty** [comment]
Fixed in 3.0.0
